docs/src/docs/asciidoc/maven-plugin-quickstart.adoc

```diff
@@ -102,7 +132,7 @@ public class Fortune {
 }
 ----
 . Copy and paste the following file,
-https://raw.githubusercontent.com/graalvm/graalvm-demos/master/fortune-demo/fortune/src/main/resources/fortunes.json[fortunes.json] under _resources/_.
```

Comment:
By convention, paths to files, directories and file names should be italicized. These are not code blocks or commands. It's better to revert to _name_ everywhere, for consistency with our documentation.